src/components/CommentDisplay.jsx

```diff
@@ -1,12 +1,31 @@
+import { useContext, useState } from "react";
+import { userContext } from "../contexts/user";
+import { deleteComment } from "./axiosrequests";
 
+const CommentList = ({ comments, setComments }) => {
+  const { user, SetUser } = useContext(userContext);
+  const [usrErr, setUsrErr] = useState(false);
+  const [isDeleting, setIsDeleting] = useState(false)
 
+  const handleDelete = (e) => {
+    setIsDeleting(true)
+    deleteComment(e.target.id)
+      .then((response) => {
+        setIsDeleting(false)
+        setComments((currComments) =>
+          currComments.filter((comment) => comment.comment_id !== Number(e.target.id)));
+      })
+      .catch((err) => {
+        if (err) {
+          setUsrErr(true);
+          setTimeout(() => setUsrErr(false), 3000);
+        }
+      });
+  };
 
-const CommentList = ({comments}) => {
-
-
- 
-
-return (
+  return (
+    <>
+    {isDeleting ? <h2>Deleting comment</h2> : null}
```

Comment:
This is good but I'm struggling to see it on your app, maybe a styling thing to look at?